### PR TITLE
Use latest renv version

### PR DESCRIPTION
**Pull request by saracastel**
# Description
This change modifies the Dockerfile to install the latest version of `renv` from GitHub instead of using a fixed version.
This was done because the pipeline build was failing locally.

# Details
#### URL to issue
N/A

#### Link to staging deployment URL (or set N/A)
N/A

#### Integration test branch
master

# Merge checklist
Your changes will be ready for merging after all of the steps below have been completed.

### Code updates
Have best practices and ongoing refactors being observed in this PR
- [ ] Migrated any selector / reducer used to the new format.

### Manual/unit testing
- [ ] Tested changes using InfraMock locally **or** no tests required for change, e.g. Kubernetes chart updates.
- [ ] Validated that current unit tests for code work as expected and are sufficient for code coverage **or** no unit tests required for change, e.g. documentation update.
- [ ] Unit tests written **or** no unit tests required for change, e.g. documentation update.

### Integration testing
**You must check the box below** to run integration tests on the latest commit on your PR branch.
Integration tests have to pass before the PR can be merged. Without checking the box, your PR
**will not pass** the required status checks for merging.

- [ ] Started end-to-end tests on the latest commit.

### Documentation updates
- [ ] Relevant Github READMEs updated **or** no GitHub README updates required.
- [ ] Relevant Wiki pages created/updated **or** no Wiki updates required.

### Optional
- [ ] Staging environment is unstaged before merging.
- [ ] Photo of a cute animal attached to this PR.

### CODE REVIEW

File: pipeline-runner/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ RUN echo ".libPaths(c('$RENV_LIB', .libPaths()))" >> $(R RHOME)/etc/Rprofile.sit
 
 # install renv to install required R packages
 RUN R -q -e "install.packages('remotes', repos = c(CRAN = 'https://cloud.r-project.org'))" && \
-    R -q -e "remotes::install_github('rstudio/renv@0.16.0')" && \
+    R -q -e "remotes::install_github('rstudio/renv')" && \
```
Comment (gerbeldo) [inline pipeline-runner/Dockerfile:23]: you should fix the version to latest here.

Comment (gerbeldo) [reply pipeline-runner/Dockerfile:23]: be mindful that they changed the way they write versions. Now it's like "v1.0.5"

Comment (gerbeldo) [reply pipeline-runner/Dockerfile:23]: Also, you will probably need to update the config files in the .renv directory

Comment (saracastel) [reply pipeline-runner/Dockerfile]: ok thanks, done

**Pull request merged**